com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -2572,51 +2571,52 @@ static int ComputeFixedPointLinearDepth(float w, float f, int numBits = 16)
         internal struct BoundedEntitySortingKeyLayout
         {
             public const int k_EntityIndexBitCount = 16;
+            public const int k_EntitySpatialKeyBitCount = 20;
 
             public int categoryBitCount;
-            public int fixedPointLogDepthBitCount;
+            public int fixedPointSpatialKeyBitCount;
             public int lightTypeBitCount;
             public int indexBitCount;
             public int totalBitCount;
 
             public int categoryOffset;
-            public int fixedPointLogDepthOffset;
+            public int fixedPointSpatialKeyOffset;
             public int lightTypeOffset;
             public int indexOffset;
         }
 
-        internal static BoundedEntitySortingKeyLayout GeBoundedEntitySortingKeyLayoutLayout()
+        internal static BoundedEntitySortingKeyLayout GetBoundedEntitySortingKeyLayoutLayout()
```

Comment:
LayoutLayout? Is the diff bugged?

Reply:
Ah, nevermind. So you fixed one typo, but then there's still another one left. :-)

Reply:
No the diff is not bugged. I checked file history and 'LayoutLayout' has been there before my change. I just changed 'Ge' to 'Get'. Should we make it single 'Layout'?

Reply:
Could you please fix my other typo as well while you are at it? Thanks!

Reply:
> Should we make it single 'Layout'?
 Yes, please.